### PR TITLE
Add onedark.nvim to made_with_lush

### PR DESCRIPTION
**Pull request by olimorris**

### CODE REVIEW

File: made_with_lush/README.md [hunk 1]
```diff
@@ -24,6 +24,22 @@ A port of the jellybeans theme.
 
 ![first screenshot](metalelf0_jellybeans-nvim_1.png)
 
+---
+### onedark.nvim
+
+[olimorris/onedark.nvim](https://github.com/olimorris/onedark.nvim)
+
```
Comment (rktjmp) [inline made_with_lush/README.md:31]: Thanks for the PR! Can you fix up this part of the readme to match the template: including the lua module name?

`require(<what you require in your colours/vim file, probably 'lush_theme.one_dark'>)`

**Review by rktjmp — CHANGES_REQUESTED** [review]

File: made_with_lush/README.md (hunk 1 above)
Comment (olimorris) [reply made_with_lush/README.md:31]: Ahhh sorry i've perhaps misinterpreted your initial ask. Have updated now.

Comment (rktjmp) [reply made_with_lush/README.md:31]: Nah, really you just made me realize that the template is missing some bits.

**Review by rktjmp — APPROVED** [review]

**Comment by rktjmp** [comment]
:tada:

**Pull request merged**